frontend/src/actions/userActions.js

```diff
@@ -98,14 +102,15 @@ export const followUser = (username) => async (dispatch, getState) => {
 
 export const updateUserProfile = (userData) => async (dispatch, getState) => {
   try {
-    dispatch({ type: UPDATE_USER_REQUEST });
+    // dispatch({ type: UPDATE_USER_REQUEST });
     let { user } = await usersService.updateUserProfile(userData);
-
+    user.profile.email = user.email;
```

Comment:
what is this for?  the API should be returning the full updated user object, we shouldn't need to change an email unless I'm misunderstanding?

Reply:
user detail reducer does not save entire user object in state it only save user.profile which does not include email so added a email property.